client/src/components/application/Find/findFilter.vue

```diff
@@ -96,149 +154,225 @@ export default {
-    //       if (sort === 'unfurnished' && tmpJSON[i].unfurnished === false) {insert = true;}
-    //       if (sort === 'approved' && tmpJSON[i].approved === false) {insert = true;}
-    //       if (sort === 'rejected' && tmpJSON[i].rejected === false) {insert = true;}
-    //
-    //       let s;
-    //       for (s = 0; s < sortedData.length; s++) {
-    //         if (sort === 'priority' || sort === 'h') {
-    //           // if (order === 'asc' || sort === 'h') {
-    //             if (tmpJSON[i].priority < sortedData[s].priority && !insert) {
-    //               sortedData.splice(s, 0, tmpJSON[i]);
-    //               insert = true;
-    //               break;
-    //             }
-    //           // } else {
-    //           //   if (tmpJSON[i].priority > sortedData[s].priority) {
-    //           //     sortedData.splice(s, 0, tmpJSON[i]);
-    //           //     insert = true;
-    //           //     break;
-    //           //   }
-    //           // }
-    //         }
-    //         if (sort === 'examiner' || sort === 'edited' || sort === 'unfurnished' || sort === 'approved' || sort === 'rejected') {
-    //           if (tmpJSON[i].examiner < sortedData[s].examiner && !insert) {
-    //             sortedData.splice(s, 0, tmpJSON[i]);
-    //             insert = true;
-    //             break;
-    //           }
-    //         }
-    //         if (sort === 'name') {
-    //           if (tmpJSON[i].name < sortedData[s].name) {
-    //             sortedData.splice(s, 0, tmpJSON[i]);
-    //             insert = true;
-    //             break;
-    //           }
-    //         }
-    //         if (sort === 'nr') {
-    //           if (tmpJSON[i].nr < sortedData[s].nr) {
-    //             sortedData.splice(s, 0, tmpJSON[i]);
-    //             insert = true;
-    //             break;
-    //           }
-    //         }
-    //       }
-    //       if (insert !== true) {
-    //         sortedData.push(tmpJSON[i]);
-    //       }
-    //     }
-    //     this.total = sortedData.length;
-    //     this.data = sortedData;
-    //   }
-    //
-    //   for (i = offset; i < offset + limit && i < sortedData.length; i++)
-    //     data.push(sortedData[i]);
-    //   this.data = data;
-    //
-    //   // this.total = data.length;
-    // },
+      } else {
+       // do nothing
+      }
+    },
+    examineNR() {
+      console.log('loading: ',this.selectedNR);
+
+      if (this.selectedNR != '') {
+        this.$store.dispatch('newNrNumber',this.selectedNR);
+        const path = '/nameExamination';
+        this.$router.push(path);
+      }
+    },
+    handleQueryChange(){
+      // let allData = this.populateTable(this.searchData);
+      this.selectedNR = '';
+      $(".select").removeClass('select');
+      let data = [];
+      let limit = this.query.limit;
+      let offset = this.query.offset;
+      // let sort = this.query.sort;
```

Comment:
General feedback, requires no code change: in general, try not to check in commented out code. It does not serve any purpose. If it's code you are changing, git will keep track of your changes. In this case it's entirely new code and console.log() calls so it definitely should not be included.